spec/services/hyrax/qa_select_service_spec.rb

```diff
@@ -1,7 +1,6 @@
 RSpec.describe Hyrax::QaSelectService do
   let(:authority) do
     # Implementing an ActiveRecord interface as required for this spec
-    # rubocop:disable RSpec/InstanceVariable
```

Comment:
This is a cop that I wish we could maintain. Can we add to Bixby?

Reply:
It is enabled here: https://github.com/samvera-labs/bixby/blob/7d62f7ce8e7ea6d22e4478dce5c8af4baddeff98/bixby_rspec_enabled.yml#L66

I think RuboCop must have made this cop less sensitive to class definitions within RSpec. This has been a pain-point for me for cases like this for some time, so I'm grateful to find that these cases are no longer flagged.

Reply:
See: https://github.com/backus/rubocop-rspec/issues/506

Reply:
Cool. 👍